src/components/ContactUsForm/index.jsx

```diff
@@ -37,6 +40,7 @@ const ContactUsForm = () => {
     }).then(() => {
       setTimeout(() => setIsSubmitted(false), 2000);
       setIsSubmitted(true);
+      setIsLoading(true);
```

Comment:
where is isLoading set to false after that?

Reply:
Thanks for reviewing, it stops spinning after post submission message pops out. I'm undecided about where to add change it's state to false again.

Reply:
Do you have any ideas where can I add it?

Reply:
Thanks @majdajroudi for helping me about setting the state to false. I have updated PR, PTAL.